torch/distributed/elastic/agent/server/local_elastic_agent.py

```diff
@@ -79,6 +80,13 @@ class LocalElasticAgent(SimpleElasticAgent):
     be propagated to the worker processes to allow them to connect to the same
     named pipe that ```LocalElasticAgent``` uses.
 
+    Logs are written to the specified log directory. Each log line will be by default
+    prefixed by "[<ROLE NAME><LOCAL_RANK>]:" string unless `log_line_prefix_template`
```

Comment:
nit:  a better way to document template pattern is by making this:

```
prefixed by ``[${role_name}${local_rank}]:`` (e.g. ``[trainer0]: foobar``). 
Log prefixes can be customized by passing a `template string <https://docs.python.org/3/library/string.html#template-strings>`_ as the ``log_line_prefix_template`` argument. 
The following macros (identifiers) are substituted at runtime: ``${role_name}, ${local_rank}, ${rank}``.
 For example, to prefix each log line with global rank instead of the local rank, set ``log_line_prefix_template = "[${rank}]:".
```